maestro/rpm/maestro.spec

```diff
@@ -112,5 +112,7 @@ install -vpm 0644 %{_filesdir}/maestro.service  %{buildroot}/%{_unitdir}
 %systemd_postun_with_restart maestro.service
 
 %changelog
+* Wed Jun 9 2021 Michael Ray <michael.ray@pelion.com> - 2.9.0-1
```

Comment:
The version here (2.9.0-1) is not consistent with the commit message (v2.10.0).

Reply:
Fixed. Thanks